include/deal.II/lac/trilinos_tpetra_vector.h

```diff
@@ -262,13 +477,42 @@ namespace LinearAlgebra
           const Number          b,
           const Vector<Number> &W);
 
+      /**
+       * A collective add operation: This function adds a whole set of values
+       * stored in @p values to the vector components specified by @p indices.
+       */
+      void
+      add(const std::vector<size_type>      &indices,
+          const std::vector<TrilinosScalar> &values);
+
+
+      /**
+       * Take an address where <tt>n_elements</tt> are stored contiguously and
+       * add them into the vector. Handles all cases which are not covered by
+       * the other two <tt>add()</tt> functions above.
+       */
+      void
+      add(const size_type  n_elements,
+          const size_type *indices,
+          const Number    *values);
+
       /**
        * Scaling and simple addition of a multiple of a vector, i.e. <tt>*this
        * = s*(*this)+a*V</tt>.
        */
       void
       sadd(const Number s, const Number a, const Vector<Number> &V);
 
+      /**
+       * This collective set operation is of lower level and can handle anything
+       * else &mdash; the only thing you have to provide is an address where all
+       * the indices are stored and the number of elements to be set.
+       */
```

Comment:
The `TrilinosWrappers::MP::Vector` class has a bunch of more overloads where this comment makes more sense. I'm not quite sure if we need all of those (or just one talking a `std::vector`?) as well but otherwise, we should adapt the comment.
 ```suggestion
       /**
        * A collective set operation: instead of setting individual elements of a
        * vector, this function allows to set a whole set of elements at once.
        * It is assumed that the elements to be set are located in contiguous memory.
        */
```